keras_retinanet/models/mobilenet.py

```diff
@@ -0,0 +1,56 @@
+import keras
+from keras import backend as K
+from keras.models import Model
+from keras.applications.mobilenet import MobileNet, BASE_WEIGHT_PATH, get_file
+
+from ..models import retinanet
+
+custom_objects = retinanet.custom_objects.copy()
+
+
+def download_imagenet(backbone):
+    alpha = 1.0
+    rows = backbone
+
+    # load weights
+    if K.image_data_format() == 'channels_first':
+        raise ValueError('Weights for "channels_last" format '
+                         'are not available.')
+    if alpha == 1.0:
+        alpha_text = '1_0'
+    elif alpha == 0.75:
+        alpha_text = '7_5'
+    elif alpha == 0.50:
+        alpha_text = '5_0'
+    else:
+        alpha_text = '2_5'
+
+    model_name = 'mobilenet_%s_%d_tf_no_top.h5' % (alpha_text, rows)
```

Comment:
Prefer to use `.format()` for formatting.

https://stackoverflow.com/questions/5082452/python-string-formatting-vs-format

Reply:
You're right!